deployment/hasura/migrations/Aerie/14_actions/up.sql

```diff
@@ -0,0 +1,188 @@
+create schema actions;
+
+DO $$
+  DECLARE seq_user text;
+  BEGIN
+    SELECT into seq_user grantee
+    FROM information_schema.role_table_grants
+    WHERE table_schema = 'sequencing'
+      AND table_name = 'user_sequence'
+      and privilege_type = 'INSERT'
+      and grantee != (select current_user as usersss)
```

Comment:
Minor: please remove the `as usersss` part. The alias isn't used, and also I just added all the s's to avoid getting distracted while working out the function, not as an acceptable alias in published code:

```suggestion
      and grantee != (select current_user)
```